HeRoBackEnd/Controllers/CandidateController.cs

```diff
@@ -14,18 +16,32 @@ public CandidateController()
           //  CandidateService candidateService = new CandidateService();
         }
 
+        /// <summary>
+        /// Returns a Json string of the list of candidates
+        /// </summary>
+        /// <returns>Json string representing a list of Candidates</returns>
+        [HttpGet]
+        [Route("Candidate/Index")]
```

Comment:
I know, but it doesn't work later. When two different controllers have methods that have the same [Route] data annotation
one of the mothods doesn't show up in the swagger. It ignores the [Route("[controller]")] part of the route. So that way I did it, it works.